src/scene/container/Container.ts

```diff
@@ -515,12 +515,15 @@ export class Container<T extends View = View> extends EventEmitter<ContainerEven
             options.view = undefined;
         }
 
-        Object.assign(this, definedProps(options));
+        assignPropertiesFromOptions(this, options, {
```

Comment:
Recommend renaming to `assignWithIgnore`. Since the functionality isn't really about props or options but a generic assign variant.

Reply:
good shout! will adjust